sapnwbootstrap-formula.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Apr 22 19:17:03 UTC 2020 - Simranpal Singh <simranpal.singh@suse.com>
+
+- Version 0.4.1
+  * Make it optional to install netweaver required packages 
+    in order to support qa mode
```

Comment:
@Simranpal I didn't see this before. Remove any reference to `qa_mode` please. This is nothing that this formula should know. The 1st sentence is good enough